t/10-jobs.t

```diff
@@ -689,12 +688,29 @@ subtest 'carry over, including soft-fails' => sub {
             $job_info = $t->app->minion->jobs({tasks => ['hook_script']})->next;
             is $job_info->{state}, 'inactive', 'hook script has been retried with long delay';
             is_deeply($job_info->{args}[2],
-                {delay => 60, retries => 1440, skip_rc => 142, kill_timeout => '30s', timeout => '5m'});
+                {delay => 60, retries => 4, skip_rc => 142, kill_timeout => '30s', timeout => '5m'});
             $notes = $job_info->{notes};
             is($notes->{hook_cmd}, 'echo delayed && exit 142;', 'real hook cmd in notes if result matches (5)');
             like($notes->{hook_result}, qr/delayed/, 'real hook cmd from config called if result matches (5)');
             is $notes->{hook_rc}, 142, 'exit code of the hook cmd is as expected';
             is $job_info->{retries}, 1, 'hook script has been retried once because of delay';
+
+            note "There is a linear delay for each hook script which gets exit 142";
+            my $linear_delay = $job_info->{delayed} - $job_info->{retried};
+            is $linear_delay, 60, 'default delay for first retry applied correctly';
+            # force some retries with delay 0 to avoid having to wait in the test
+            $t->app->minion->job($job_info->{id})->retry({delay => 0});
+            $t->app->minion->job($job_info->{id})->retry({delay => 0});
```

Comment:
In os-autoinst we use Test::Mock::Time so try if that helps here as well

Reply:
And in any case, this line seems to be duplicated.

Reply:
@b10n1k Please remove the duplicate line

Reply:
Or use a loop to avoid the duplication.